package.json

```diff
@@ -5,10 +5,30 @@
   "author": "Valera <Koniushok@users.noreply.github.com>",
   "private": true,
   "dependencies": {
-    "webpack": "^4.28.3",
-    "webpack-cli": "^3.2.1"
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
+    "styled-components": "^4.1.3"
   },
   "scripts": {
+    "start": "webpack-dev-server  --mode development  --open ",
```

Comment:
2 spaces inside script commands? Was it done by guides or by mistake?

Reply:
ready